Assets/Scripts/UI/Menus/OptionsMenu.cs

```diff
@@ -4,10 +4,6 @@
 
 namespace OperationBlackwell.UI {
 	public class OptionsMenu : Menu {
-		public void AudioClicked() {
```

Comment:
Was this only removed, and how do we enter the audio menu now?

Reply:
It was a placeholder, we now set the correct gameobject to active and the main options menu to inactive, but that happens with events and is set up within the editor.